parquet-scrooge/pom.xml

```diff
@@ -98,7 +98,7 @@
     <dependency>
       <groupId>com.twitter</groupId>
       <artifactId>scrooge-core_${scala.binary.version}</artifactId>
-      <version>4.7.0</version>
+      <version>19.1.0</version>
```

Comment:
@xhochy could you please upgrade scrooge version to 19.9.0? It includes a [fix](https://github.com/twitter/scrooge/commit/e6ac7e1304112bb8b10f9c127a6697e65d14fbec) for the String type related issue which we experience in the latest build.